app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Uh, I didn’t notice it. EAPI=8

Reply:
EAPI=7 is ok for now as we do not use EAPI 8 features and 7 is still supported.

Reply:
Which one does not have EAPI=8? Both [linux-mod](https://github.com/gentoo/gentoo/blob/master/eclass/linux-mod.eclass#L10) and [toolchain-funcs](https://github.com/gentoo/gentoo/blob/master/eclass/toolchain-funcs.eclass#L7) have it enabled?